blog/README.md

```diff
@@ -0,0 +1,57 @@
+# Knative blog
+
+The Knative blog is owned by the [Documentation working group](https://knative.dev/community/contributing/working-groups/working-groups/#documentation) and run by the [Editorial Team](#leadership).
+
+This section covers documentation, processes, and roles for the [Knative blog](https://knative.dev/blog/).
+
+## Leadership 
+
+- **Technical Editors:** [Lionel Villard](https://github.com/lionelvillard), [Evan Anderson](https://github.com/evankanderson)
+- **Copy Editors:** [Ashleigh Brennan](https://github.com/abrennan89), [Caroline Lee](https://github.com/carieshmarie), [María Cruz](https://github.com/macruzbar)
+- **Blog Community Managers:**  [María Cruz](https://github.com/macruzbar), [Jonatas Baldin](https://github.com/jonatasbaldin)
+
+## Contact
+
+- Slack: [#docs](https://knative.slack.com/archives/C9CV04DNJ)
+
+## Submit a Post
+
+Anyone can write a blog post and submit it for review. Commercial content is not allowed. Please refer to the [blog guidelines](#blog-guidelines) for more guidance.
+
+To submit a blog post, follow the steps below.
+
+1. [Sign the Contributor License Agreements](https://github.com/knative/community/blob/master/CONTRIBUTING.md#contributor-license-agreements) if you have not yet done so.
+1. Familiarize yourself with the Markdown format for existing blog posts in the [docs repository](https://github.com/knative/docs/tree/master/blog). Blog posts are categorized into different directories. You can explore the directories to find examples.
+1. Write your blog post in a text editor of your choice.
+1. (Optional) If you need help with markdown, check out [StakEdit](https://stackedit.io/app#) or read [GitHub's formatting syntax](https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax) for more information. 
+1. Choose a directory in the [docs repository](https://github.com/knative/docs/tree/master/blog), and click **Create new file**.
```

Comment:
Should the text of the link be "in the blog directory"?

Reply:
I think it's redundant